### PR TITLE
🏃 Add kustomizeconfig for namespace

### PR DESCRIPTION
**Pull request by furkatgofurov7**
**What this PR does / why we need it:**
This PR goes together with [#PR68](https://github.com/metal3-io/cluster-api-provider-metal3/pull/68)

### CODE REVIEW

**Comment by maelk** [comment]
@furkatgofurov7 
This is not tested by the CI. Did you test that manually ?

File: deploy/default/kustomization.yaml [hunk 1]
```diff
@@ -8,6 +8,10 @@ resources:
 - ../operator
 - ../../ironic-deployment/default
 configMapGenerator:
+- behavior: replace
 - envs:
```
Comment (maelk) [inline deploy/default/kustomization.yaml:12]: you have two "-" you should only have the first one

**Comment by furkatgofurov7** [comment]
test-integration

**Pull request merged**